main/guides/agoric-cli/agd-query-tx.md

```diff
@@ -0,0 +1,167 @@
+# Using `agd` to make queries and transactions
+
+`agd` is the Agoric Cosmos App, analagous to `simd` in the [Cosmos SDK](https://docs.cosmos.network/) simapp or `gaiad` in
+the Cosmos hub. Most of the `simd` [query commands](https://docs.cosmos.network/v0.46/core/cli.html#query-commands) and [transaction commands](https://docs.cosmos.network/v0.46/core/cli.html#transaction-commands) work similarly in `agd`.
+
+::: tip agd for Building Dapps
+
+This section focusses on commands relevant to developing and deploying smart contracts and dapps. See also:
+
+- [Validators topics \- Agoric Community Forum](https://community.agoric.com/c/validators/9)
+- [Governance topics \- Agoric Community Forum](https://community.agoric.com/c/governance/6)
+- [Delegator Guide \(CLI\) \| Cosmos Hub](https://hub.cosmos.network/main/delegators/delegator-guide-cli.html)
+
+:::
+
+::: tip Installing agd
+
+Options include:
+
+- Use the [basic dapp local chain](../getting-started/#starting-a-local-agoric-blockchain) docker container: to run `agd status`, enter `yarn docker:bash` followed by `agd status`; or use `docker-compose exec agd agd status`.
```

Comment:
> So for me, yarn `docker:bash` gave some errors, I had to run `yarn start:docker` first for it to work.

Were the errors / diagnostics straightforward to understand? You evidently figured out how to proceed. It's not practical to make `yarn docker:bash` work without `yarn start:docker` first, but if the diagnostics aren't clear, we could perhaps address that.

Reply:
re separate repo: it's worth considering

 - https://github.com/Agoric/documentation/issues/902

Reply:
Re errors / diagnostics, I was getting a pretty generic error

```
Error: post failed: Post "http://localhost:26657": dial tcp 127.0.0.1:26657: connect: connection refused
```

And I figured it out by backtracking my way through `Your First Agoric Dapp` section and run the docker related command preceding `yarn docker:bash`

Reply:
you got that error in response to `yarn docker:bash`? that seems unlikely. Perhaps in response to `agd status`? That's possible, but it would mean you have `agd` available outside the container.